src/app/beer_garden/api/http/__init__.py

```diff
@@ -184,89 +295,11 @@ def _setup_application():
 
 
 def _setup_tornado_app() -> Application:
-    prefix = config.get("entry.http.url_prefix")
```

Comment:
This is a good refactor regardless of the motivation because it makes the code cleaner.